### ISSUE TITLE
Status changes on NPCs

### ISSUE THREAD

**Issue by Lemongrass3110** [rathena/rathena#1734]
The clan leader NPCs all have the status change SC_CLANINFO with their respective clan id to show the clan icon just like on players below them.

For players we currently have visible status changes that need to be sent out(`clif.c::clif_efst_status_change_sub`) to other players stored in the map_session_data in the following fields:
```
struct sc_display_entry **sc_display;
unsigned char sc_display_count;
```

This is required for NPCs now too.

**Comment by cydh** [comment]
oh u talked about this back then, it's fine by me. add those 2 vars to `struct npc_data`, but don't forge the `eri *pc_sc_display_ers;`

**Comment by aleos89** [comment]
Implemented in 45e91e2.
